Formula/ignition-launch0.rb

```diff
@@ -29,8 +29,7 @@ def install
     system "make", "install"
   end
 
-  # TODO: fix test. Failing: https://build.osrfoundation.org/job/generic-release-homebrew_bottle_builder/209/label=osx_mojave/
-  # test do
-  #  system "ignition", "-run", "config/gazebo.ign"
-  # end
+  test do
+    system "ignition", "-run", "config/gazebo.ign"
```

Comment:
see https://bitbucket.org/ignitionrobotics/ign-launch/issues/2/should-example-launch-ign-files-be and https://bitbucket.org/ignitionrobotics/ign-launch/issues/3/example-command-in-readme-is-out-of-date